plugins/org.komodo.eclipse.workspace.ui/src/org/komodo/eclipse/workspace/ui/common/util/ICredentialsCommon.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
This should either be part of an existing spi interface, part of a utils class or added to util plugin